tests/RecipeTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Tests;
+
+use Composer\Package\Package;
+use PHPUnit\Framework\TestCase;
+use Symfony\Flex\Recipe;
+
+class RecipeTest extends TestCase
+{
+    /**
+     * @var Recipe
+     */
+    private $recipe;
+
+    /**
+     * @var Package|PHPUnit_Framework_MockObject_MockObject
```

Comment:
I think, it should be : `@var Package|PHPUnit\Framework\MockObject\MockObject`

Reply:
Actually, it should be `@var Package|\PHPUnit\Framework\MockObject\MockObject` (note the missing leading `\`).